.jshintrc

```diff
@@ -23,7 +23,7 @@
   "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
   "unused"        : false,    // true: Require all defined variables be used
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
-  "maxparams"     : 10,       // {int} Max number of formal params allowed per function
+  "maxparams"     : 12,       // {int} Max number of formal params allowed per function
```

Comment:
@AllenBW Don't know if you already received the github link to the offending line or not, but here it is anyway --
https://github.com/ZitaNemeckova/manageiq-ui-self_service/blob/7625be60e7942d5b5e15ba57919e6bbd2443553d/client/app/states/login/login.state.js#L29

And you know what I just noticed? `CollectionsApi` ngInject is not being used at all. Does that mean we caught a lucky break? 😄 
We could just delete `CollectionsApi` and then the params count goes back to 10, which takes care of the offense (for now)